applications/nrf_desktop/sample.yaml

```
@@ -171,3 +171,18 @@ tests:
       - nrf52840dongle/nrf52840
     tags: bluetooth ci_build
     extra_args: CONF_FILE=prj_release_4llpmconn.conf
+  applications.nrf_desktop.zdebug_nrf54h20:
+    build_only: true
+    sysbuild: true
+    platform_allow: >
+      nrf54h20dk/nrf54h20/cpuapp
+    integration_platforms:
+      - nrf54h20dk/nrf54h20/cpuapp
+  applications.nrf_desktop.zrelease_nrf54h20:
+    build_only: true
+    sysbuild: true
+    platform_allow: >
+      nrf54h20dk/nrf54h20/cpuapp
+    integration_platforms:
+      - nrf54h20dk/nrf54h20/cpuapp
+    extra_args: FILE_SUFFIX="release"
```

You don't really need to use quotation marks

```suggestion
    extra_args: FILE_SUFFIX=release
```